docs/tutorials/TracingCode.md

```diff
@@ -231,7 +231,7 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
      * {@code JsonSerializableAddressBook}.
      */
     public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
```

Comment:
Should the JsonSerializableAddressBook(ReadOnlyAddressBook source) be changed to JsonSerializableGuestBook(ReadOnlyGuestBook source)?